ruby/ql/test/query-tests/security/cwe-022/ArchiveApiPathTraversal.rb

```diff
@@ -0,0 +1,69 @@
+class TestContoller < ActionController::Base
```

Comment:
@aibaars I moved the tests in their entirety to the test directory for the existing Path Traversal query to prove that 1. it would pick up the use of `File.open()` within `TarReader` as expected (it does) and 2. that it would find uses of `Zip::File.open()` now that it's been added as a model extending `FileSystemAccess`. I'm happy to get rid of this file and fold them into the existing test file if that works better.